src/base/static/components/input-form/index.js

```diff
@@ -349,11 +345,27 @@ class InputForm extends Component {
         attrs[field.name] = extractEmbeddedImages(attrs[field.name]);
       });
 
-    // Fire pre-save hook.
-    // The pre-save hook allows flavors to attach arbitrary data to the attrs
-    // object before submission to the database.
-    if (this.props.customHooks && this.props.customHooks.preSave) {
-      attrs = hooks[this.props.customHooks.preSave](attrs);
+    // Run geospatial analyses:
+    if (this.selectedCategoryConfig.geospatialAnalysis) {
```

Comment:
For the time being all of the configuration for geospatial analysis lives in the "category" section of the `place_detail` config. I'm sure this will change as we port over forms to the backend, but I think we can update the frontend to accommodate changes as needed.